src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -122,9 +124,22 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
 
             URL wmsUrl = new URL(wmsUrlStr + "?" + queryString);
 
-            IOUtils.copy(wmsUrl.openConnection().getInputStream(), response.getOutputStream());
+            HttpURLConnection connection = (HttpURLConnection)wmsUrl.openConnection();
```

Comment:
Looks good @pmbohm. I'm wondering if there is any issue that the connection won't get closed, if any of the code below throws an exception?